task/09-functions-n-closures-tasks.js

```diff
@@ -65,7 +69,18 @@ function getPowerFunction(exponent) {
  *   getPolynom()      => null
  */
 function getPolynom() {
-    throw new Error('Not implemented');
+    let ln = arguments.length;
+
+    switch (ln) {
+        case 1:
+            return (x) => arguments[0];
+        case 2:
+            return (x) => (arguments[0] * x + arguments[1]);
+        case 3:
+            return (x) => (arguments[0] * Math.pow(x, 2) + arguments[1] * x + arguments[2]);
+        default:
+            return () => null;
```

Comment:
This solution works for arguments count less or equal three
Please, find more universal solution.
It should have some kind of loop